docker/openemr/flex/Dockerfile

```diff
@@ -122,6 +122,8 @@ COPY utilities/devtoolsLibrary.source /root/
 RUN mkdir /snapshots
 RUN mkdir /certs
 RUN mkdir -p /couchdb/original
+#Copy auto_configure so we can do backup/restore with dev tools
```

Comment:
also clarify in comment that we are aware are also adding above (line 113) to another place (just so we don't remove one of them in future)

Reply:
rec removing lines 432-433 in openemr.sh to clean up loose end